x/fundraising/spec/01_concepts.md

```diff
@@ -4,18 +4,71 @@
 
 ## Auction Types
 
-There are two different types of auctions in the fundraising module.
+This fundraising module provides two different types of auctions: 1) Fixed Price Auction and 2) Batch Auction.
 
-### Fixed Price Auction
+## Fixed Price Auction
 
-A fixed price auction type is the most basic type of an auction that is first come first served way to raise funds. When an autioneer creates this fixed price auction, they must determine the fixed start price that is proportional to the paying coin denom. Once it is created, bidders can only bid with the start price of the auction. For example, an auctioneer sets 0.5 as start price for X coin and the paying coin is Y coin. How many X coins does a bidder receives if they bids 10Y coin? It is calculated as Y coin over start price (10/0.5), which results to 20X coin.  
+This fixed price auction is to sell a given amount of tokens on a first-come, first-served basis.
 
-### English Auction
+### What an auctioneer does:
+When an auctioneer creates a fixed price auction, it must determine the following parameters.
 
-An english auction type is an ascending dynamic auction that an auctioneer decides the starting price of the selling amount of coin and bidders bid to purchase the amounts of the coin. Competition for the price, not how fast bidders can bid.
+- **Selling Token**: the denom of tokens to be auctioned,
+- **Paying Token**: the denom of tokens to be used for payment,
+- **Price**: fixed amount of the paying tokens to get a selling token (i.e., amount of paying tokens per selling token),
+- **Auction Start Time**: when the auction starts,
+- **Auction End Time**: when the auction ends,
+- **Offering Quantity**: total amount of selling tokens to be auctioned.
 
-## Extended Auction Round(s)
+### What bidders can/cannot do:
+
+A bidder can place a new bid with a fixed amount of paying tokens. 
+A bidder cannot modify or cancel the existing bid it previously placed.
+
+### When the auction ends:
+
+The auction will end either when Auction End Time is arrived or when the entire Offering Quantity is sold out.
+
+
+
+## Batch Auction
+
+This batch auction allows each bidder to participate in the auction by placing limit orders with a bid price chosen freely at any time within the auction period. An order book is created to record the bids with various bid prices.
+
+### What an auctioneer does:
+
+When an auctioneer creates this batch auction, it must determine the following parameters.
+
+- **Selling Token**: the denom of tokens to be auctioned,
+- **Paying Token**: the denom of tokens to be used for payment,
+- **Auction Start Time**: when the auction starts,
+- **Auction End Time**: when the auction ends,
+- **Offering Quantity**: total amount of selling tokens to be auctioned.
```

Comment:
Same comment as above. I suggest to use names to be the exact names that are used in codebase and an auctioneer needs to input `Vesting Schedules`.